arches_lingo/src/arches_lingo/constants.ts

```diff
@@ -28,3 +28,10 @@ export const ENGLISH = {
     name: "English",
     scope: "system",
 };
+
+export const LANGUAGE_CONTROLLED_LIST = "55ce793b-a51a-4b25-811d-d08ea797f8c3";
```

Comment:
IMO hardcoding URIs on the frontend isn't a preferred pattern. Is there a way to get these supplied by the backend? If not or if it becomes super complex then okay, but IMO this pattern should be avoided if possible.